Change exportToPDF demo descriptions (#1771)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/PDFHeaderAndFooter/description.md

> The DataGrid allows to customize a header (a title before exported data) and a footer (a note after exported data) in the exported PDF file. Follow the steps below to configure a header/footer:

Reviewer: ```suggestion
The DataGrid allows you to customize a header and a footer in the exported PDF file. Follow the steps below to configure a header/footer:
```

1. Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method.

2. Configure the following properties:    
    - For a header   
    Use the [topLeft](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/topLeft/topLeft.md) object to specify a start position for DataGrid export. This position should be below the header.
    - For a footer    

>     Use the [customDrawCell](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customDrawCell) function to calculater the coordinate of the last right cell of the DataGrid.

Author: ```suggestion
    Use the [customDrawCell](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customDrawCell) function to calculate the coordinate of the last right cell of the DataGrid.
```

>     Use the [customDrawCell](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customDrawCell) function to count the coordinate of the last right cell of the DataGrid.

Author: ```suggestion
    Use the [customDrawCell](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customDrawCell) function to calculate the coordinate of the last right cell of the DataGrid.
```

<a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a> library allows you to customize PDF files, write custom texts and draw custom content. This demo uses this functionality to add a header (a title before exported data) and a footer (a note after exported data).

## Add a Header

1. Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method.

> 2. Specify the [topLeft](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/topLeft/topLeft.md) object to set a starting position for DataGrid export. This position needs to be below the header.

Reviewer: ```suggestion
2. Use the [topLeft](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/topLeft/topLeft.md) object to specify a start position for DataGrid export. This position should be below the header.
```
